manifests/charts/istio-cni/values.yaml

```diff
@@ -44,6 +44,12 @@ cni:
   # Allow the istio-cni container to run in privileged mode, needed for some platforms (e.g. OpenShift)
   privileged: false
 
+  # Configure ambient settings
+  ambient:
+    # If enabled, ambient redirection will be enabled
+    enabled: false
```

Comment:
Same as the other review - eventually this should move to a configmap outside of values.yaml/install realm.